docs/electron.rst

```diff
@@ -121,6 +125,34 @@ pulseaudio for sound and enables network access:
     - --socket=pulseaudio
     - --share=network
 
+.. note::
+
+  Native wayland support in electron experimental and often unstable.
+  It is advised to stick with the X11 and Xwayland configuration above
+  as the default.
+
+To enable experimental `native Wayland` support in electron, the following
+flags can be passed to the program depending on the Electron version it
+uses.
+
+- ``--ozone-platform-hint=auto`` for Electron>=20. `auto` will
+  choose wayland when the seesion is wayland and x11 otherwise. With
+  this, to make `native wayland` the default for users ``--socket=fallback-x11``
+  and ``--socket=wayland`` must be used.
+
+  The recommended option is to leave it to the user. So ``--socket=x11``
+  can be used in manifest and Wayland can be tested with
+  ``flatpak run --socket=wayland org.flathub.electron-sample-app``.
+
+- ``--enable-features=UseOzonePlatform --ozone-platform=wayland`` for
```

Comment:
I might just leave out this information for very old Electron versions personally.

Reply:
Ok sure. I kept it because flathub has a few apps using very old electron versions